pom.xml

```diff
@@ -47,6 +47,11 @@
             <version>1.2</version>
         </dependency>
         <dependency>
+            <groupId>io.jenkins.blueocean</groupId>
+            <artifactId>blueocean-rest</artifactId>
+            <version>1.1.0-beta-2-SNAPSHOT</version>
```

Comment:
It may blow up at any moment since the plugin depends on higher Core version. So it may be missing some API